src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/PrisonerService.kt

```diff
@@ -203,6 +204,7 @@ class PrisonerService(
 
     val defaultPathwayStatuses = getDefaultPathwayStatuses()
     val watchedOffenders = watchlistService.findAllWatchedPrisonerForStaff(staffUsername)
+    val assignedWorkers = caseAllocationService.getAllAssignedRessettlementWorkers()
```

Comment:
Should query based on the prison the search is for. This could eventually get very big if we're pulling every allocated case from every prison for each search

Reply:
done